mars/assetlist.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "mars",
+  "assets": [
+    {
+      "description": "Mars protocol token",
+      "denom_units": [
+        {
+          "denom": "umars",
+          "exponent": 0,
+          "aliases": ["micromars"]
+        },
+        {
+          "denom": "mars",
+          "exponent": 6,
+          "aliases": ["mars"]
```

Comment:
Worried about including an alias that is identical to the denom value. Can we remove the aliases array for this denom?

Reply:
Good point. I removed the aliases (removed “micromars” as well, as I don’t think anyone would use it in practice.)